rxandroid/src/main/java/io/reactivex/android/schedulers/AndroidSchedulers.java

```diff
@@ -15,13 +15,26 @@
 
 import android.os.Handler;
 import android.os.Looper;
+
+import java.util.concurrent.Callable;
+
 import io.reactivex.Scheduler;
 import io.reactivex.android.plugins.RxAndroidPlugins;
 
 /** Android-specific Schedulers. */
 public final class AndroidSchedulers {
+
+    private static final class MainHolder {
+
+        static final Scheduler DEFAULT = new HandlerScheduler(new Handler(Looper.getMainLooper()));
+    }
+
     private static final Scheduler MAIN_THREAD = RxAndroidPlugins.initMainThreadScheduler(
-        new HandlerScheduler(new Handler(Looper.getMainLooper())));
+            new Callable<Scheduler>() {
+                @Override public Scheduler call() throws Exception {
+                    return MainHolder.DEFAULT;
```

Comment:
That's fine. Let's leave it. The indirection ensures that the classes aren't even loaded let alone run if you never call `call()`.